edx_exams/apps/core/tests/test_api.py

```diff
@@ -569,6 +569,7 @@ def setUp(self):
             exam_name='test_exam',
             exam_type='proctored',
             time_limit_mins=30,
+            is_active=True,
```

Comment:
could you throw another exam in these tests with is_active=false just to get some test coverage?

Reply:
done!